stacks_queues/queue.py

```diff
@@ -17,45 +17,65 @@ def __init__(self):
         self.size = 0
       
 
+    # methods to add to the Queue class
     def enqueue(self, element):
         """ Adds an element to the Queue
             Raises a QueueFullException if all elements
             In the store are occupied
             returns None
         """
-        pass
```

Comment:
✨

Reply:
resubmitted, some formatting was off for questions, and missing a couple answers